### DISCUSSION TITLE
plugins did not run

### DISCUSSION THREAD

**Discussion by JXZ201606** [rabbitmq/rabbitmq-delayed-message-exchange#223] [General]
### Describe the bug

I installed the rabbitmq plug-in with k8s, and then started it. The startup was successful, but the plug-in did not run

### Reproduction steps

1.
Put the plugin into the /plugins folder
2.
Start the plugin
![image](https://user-images.githubusercontent.com/40420927/228111502-df9db4ca-be9f-4493-b35f-acb8242a56ee.png)
3.
The plugin starts successfully but does not run
...
![image](https://user-images.githubusercontent.com/40420927/228111653-49544973-0f02-4433-926c-dad36b0e1bf9.png)

### Expected behavior

How to make the plugin work

**Comment by michaelklishin** [comment]
Enabling the plugin will give you a new exchange type to use. The actual delaying happens when you declare such an exchange and begin publishing messages to it.

FYI, RabbitMQ 3.9 is [out of support](https://rabbitmq.com/versions.html), so consider upgrading.
